admin/workspace-providers.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Workspace Providers
+description: Learn how workspace providers can minimize latency for users.
+state: beta
+---
+
+Coder allows you to add to your deployment multiple Kubernetes clusters. These
+additional clusters are called **providers**. This can be helpful for
+distributed teams, since users can choose the provider closest to them to
+minimize latency.
+
+## Enabling Workspace Providers
+
+To enable the Workspace Providers UI elements:
+
+1. In the Coder UI, click on your avatar.
```

Comment:
I was thinking of creating a page called "How to Enable Feature Previews" and just link to it from all the instances where we write it out?

Reply:
That seems good too. I just generally think if we can make direct links to particular places in the frontend, it's more convenient for the person reading the documentation - rather than looking around for buttons, they can go directly where they need to go.

That said, this would need to be on top of the existing instructions (not a replacement for them) since they might want to know how to find the feature normally without using the link from the documentation.

Reply:
> though I don't know if that's currently possible

Unfortunately it's currently not, though I agree that it ought to be, which is part of a larger Next/page paradigm refactor that's currently underway :)

However (and I left this comment below as well) - that flag is internal and will remain internal for 1.16